jekyll/_includes/ja/global-nav.html

```diff
@@ -1,161 +1,63 @@
-                <li class="subnav-item"><a href="{{ '/open-source/' | outer_url }}">オープンソース</a></li>
-              </ul>
-            </li>
-            <li class="arrow collapsed border-bottom">
-              <span class="heading">
-                <i class="fa fa-angle-right" aria-hidden="true"></i>
-                会社概要
-              </span>
-              <ul class="subnav">
-                <li class="subnav-item"><a href="{{ '/about/' | outer_url }}">CircleCI について</a></li>
-                <li class="subnav-item"><a href="{{ '/customers/' | outer_url }}">ケース スタディー</a></li>
-                <li class="subnav-item"><a href="{{ '/blog/' | outer_url }}">ブログ</a></li>
-                <li class="subnav-item"><a href="{{ '/careers/' | outer_url }}">採用情報</a></li>
-                <li class="subnav-item"><a href="{{ '/contact/' | outer_url }}">お問い合わせ</a></li>
-                <li class="subnav-item"><a href="{{ '/partners/' | outer_url }}">パートナー募集</a></li>
-              </ul>
-            </li>
-            <li class="arrow collapsed border-bottom">
-              <i class="fa fa-angle-right" aria-hidden="true"></i>
-              サポート
-              <ul class="subnav">
-                <li class="subnav-item"><a href="https://academy.circleci.com/?access_code=public-2020" target="_blank">トレーニング</a></li>
-                <li class="subnav-item"><a href="https://support.circleci.com/ja" target="_blank">サポート情報</a></li>
-                <li class="subnav-item"><a href="https://status.circleci.com/" target="_blank">システム稼働状況</a></li>
-                <li class="subnav-item"><a href="https://ideas.circleci.com/" target="_blank">改善要望</a></li>
-                <li class="subnav-item"><a href="{{ '/support/premium-support/' | outer_url }}">プレミアム サポート</a></li>
-              </ul>
-            </li>
-          </ul>
+          <div class="global-nav--search-bar">
+            {% include main-searchbar.html %}
+          </div>
+        </div>
+      </div>
+      <nav class="collapse global-nav--nav" id="global-nav">
+        <div class="divider"></div>
+        {% include_cached mobile-sidebar.html site=site page=page %}
+        <div class="global-nav--list global-nav--external-links">
+          <a
+            href="{{ '/' | outer_url }}"
+            class="global-nav--external-link global-nav--list-item"
+            data-analytics-action="click-outer-cta"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            circleci.com&nbsp;&nbsp;{% include utility-icons/external.svg %}
+          </a>
+        </div>
+        <div class="global-nav--footer">
+          <a
+            href="{{ '/signup/' | outer_url }}"
+            class="btn-primary visitor-item"
+            target="_blank"
+            data-analytics-action="click-outer-cta"
+          >
+            無料でビルドを開始する
+          </a>
+          <a href="/dashboard" class="btn-primary dashboard-link">
```

Comment:
I believe this has to be `circleci.com/dahboard` (or `https://app.circleci.com/pipelines` )? as this will link to `/docs/dashboard`; which is what is causing the build to fail with the html-proofer gem.

Reply:
I updated this to `https://app.circleci.com/dashboard` on the preview branch